From: Paul Boddie <paul@boddie.org.uk>
Date: Tue, 15 Aug 2023 23:50:33 +0200
Subject: Add contrib package data for intermap.txt.

The moin create-instance command fails if intermap.txt is not installed.

moin (2.0+git20230330.c55318e3-1) unstable; urgency=medium

  * Initial release.
---
 setup.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/setup.py b/setup.py
index 16217de..93fcf7e 100644
--- a/setup.py
+++ b/setup.py
@@ -62,6 +62,7 @@ setup_args = dict(
         'moin.templates': ['*.html', '*.xml', ],
         'moin.apps.admin.templates': ['*.html', ],
         'moin.apps.misc.templates': ['*.html', '*.txt', ],
+        'moin.contrib': ['*.txt', ],
     },
     include_package_data=True,
     zip_safe=False,
